internal/storage/postgres/mapper/vehicle_test.go

```diff
@@ -95,5 +103,44 @@ var allTestVehicles = []*sqlc.Vehicle{
 		NumberPlate:   "FL TZ 1235",
 		Description:   "This is a small car",
 		WaterCapacity: 1000,
+		Type:          sqlc.VehicleTypeTransporter,
+		Status:        sqlc.VehicleStatusNotavailable,
 	},
 }
+
+func TestMapVehicleStatus(t *testing.T) {
+	tests := []struct {
+		input    sqlc.VehicleStatus
+		expected entities.VehicleStatus
+	}{
+		{input: sqlc.VehicleStatusActive, expected: entities.VehicleStatusActive},
+		{input: sqlc.VehicleStatusAvailable, expected: entities.VehicleStatusAvailable},
+		{input: sqlc.VehicleStatusNotavailable, expected: entities.VehicleStatusNotAvailable},
+		{input: sqlc.VehicleStatusUnknown, expected: entities.VehicleStatusUnknown},
+	}
+
+	for _, test := range tests {
+		t.Run(string(test.input), func(t *testing.T) {
```

Comment:
To have all tests the same, I think it would be better to have here t.Run("should return etc"